frontend/src/app/modules/home/dashboard/dashboard.component.ts

```diff
@@ -275,7 +275,16 @@ export class DashboardComponent extends BaseComponent implements OnInit, OnDestr
     }
 
     private revertTilesOrder() {
-        this.tiles = this.tiles.sort(t => t.tileOrder);
+        this.tiles = this.tiles.sort((t1, t2) => t1.tileOrder - t2.tileOrder);
```

Comment:
Please move it to a separate method (in a separate file with utils)

Reply:
Fixed